apps/services/auth/ids-api/src/app/user-profile/user-profile.service.ts

```diff
@@ -45,7 +47,14 @@ export class UserProfileService {
     private companyRegistryApi: CompanyRegistryClientService,
     @Inject(LOGGER_PROVIDER)
     private logger: Logger,
-  ) {}
+    @Optional()
+    readonly configService?: ConfigService,
+  ) {
+    if (configService) {
+      this.alsoUseFakeApi =
+        configService.get<boolean>('ALSO_USE_FAKE_USER_API') ?? false
+    }
```

Comment:
You could do and remove L#34
```suggestion
      this.alsoUseFakeApi =
        configService?.get<boolean>('ALSO_USE_FAKE_USER_API') ?? false
```

Reply:
Fixed.